minifi-c2/minifi-c2-commons/src/main/java/org/apache/nifi/minifi/c2/model/Device.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.minifi.c2.model;
+
+public class Device extends DeviceInfo {
+
+    private String name;
+    private String firstSeen;
```

Comment:
what do first seen and last seen mean in this context? I assume they are timestamps?

Reply:
Yes, they should be timestamps. I meant to ask this question, so I'll just ask this here: what should the preferred JSON format of timestamps be for the C2 server REST API? We can go with Epoch time or another standard representation such as [RFC 3339](https://tools.ietf.org/html/rfc3339), which is a stricter version of ISO 8601, e.g., 2018-03-19T17:51:00+00:00. If Epoch time, this field would change to a long of course, but should the units be seconds or milliseconds since epoch?